book_haven_api/tests/requests/libraries/books/test_add_book_to_library.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from django.urls import reverse
+from rest_framework.test import APIClient
+from book_haven_api.models import Library, Book
+
+@pytest.fixture
+def Libraries():
+    library1 =  Library.objects.create(
+        id=1,
+        name='Library1',
```

Comment:
I'd highly recommend switching to `Faker` to generate this type of information. This can also expose issues in your application, in things like frontend E2E tests, when you generate an actual name and it doesn't display correctly, etc. You might also want to eventually do validations on things like names, and having more control using `Faker` is almost always a better path forward.